docs/linux-tricks/handy-cli-apps.md

```diff
@@ -7,39 +7,39 @@ date: 2021-10-31
 
 ### 简介
 
-Homebrew 是一款起源于服务 macOS 的包管理器，特色是在用户态进行包管理和安装，做法非常整齐优雅，并且有非常大的社区支持。虽说 Linux 下各发行版也有用很多包管理器，但是总是会缺少一些好用的小软件，这时 Homebrew 就有用武之地了。强烈推荐先安装 Homebrew，因为可能会更方便地安装下面介绍的软件。
+Homebrew 是一款起源于服务 macOS 的包管理器，特色是在用户态进行包管理和安装，做法非常整齐优雅，并且有非常大的社区支持。虽说 Linux 下各发行版也有用很多包管理器，但是总是会缺少一些好用的小软件，这时 Homebrew 就有用武之地了。强烈推荐先安装 Homebrew，因为可能会更方便地安装下面介绍的软件 ~~指下面的软件全部都能用 `brew install` 安装~~。
 
 ### 资源
 
-- 官方网站、快速入门：[https://brew.sh](https://brew.sh)
-- Homebrew TUNA 镜像源配置教程：[https://mirrors.tuna.tsinghua.edu.cn/help/homebrew/](https://mirrors.tuna.tsinghua.edu.cn/help/homebrew/)
+- 官方网站、快速入门：<https://brew.sh>
+- Homebrew TUNA 镜像源配置教程：<https://mirrors.tuna.tsinghua.edu.cn/help/homebrew/>
 
-## tldr [![tldr homebrew version](https://img.shields.io/homebrew/v/tldr?style=flat-square)](https://formulae.brew.sh/formula/tldr)
+## tldr [![tldr status](https://repology.org/badge/tiny-repos/tldr.svg)](https://repology.org/project/tldr/versions)
 
 ### 简介
 
 Too long; didn't read. ~~即 “太长不看”~~ 废话不多说，就是这款小工具的目的。想必各位时不时都要用一些自己没用过的命令，做一些简单的事情，但却要翻看天书一般 man，可谓杀鸡用牛刀。tldr 提供的文档则恰恰相反：直击痛点，通俗易懂，示例清晰。
 
 ### 资源
 
-- 官方网站：[https://tldr.sh](https://tldr.sh)
-- 快速入门：安装 Homebrew 后运行 `brew install tldr` 安装。用法与 `man` 类似，在想要查询的命令前加上 `tldr`，比如查询 `tar` 的用法就运行 `tldr tar`。
-- 其他实现：~~因为是使用 NodeJS 实现~~，官方的实现很慢。可以使用 Rust 实现 tealdeer 来代替。 [![tealdeer homebrew version](https://img.shields.io/homebrew/v/tealdeer?style=flat-square)](https://formulae.brew.sh/formula/tealdeer)
-- 相关中文教程：[https://learnku.com/articles/23834](https://learnku.com/articles/23834)
+- 官方网站：<https://tldr.sh>
+- 快速入门：用法与 `man` 类似，在想要查询的命令前加上 `tldr`，比如查询 `tar` 的用法就运行 `tldr tar`。
```

Comment:
I don't think it would be a good "quick start" if without installation guide.

Reply:
Few apart from Mac users actually use `brew` in daily drive. Pages for package names in different repositories (which can be reached by clicking badges) are more practical than `brew install xxx`. Anyway, a tip for that will be added.

Reply:
141e52f

Reply:
141e52f